### ISSUE TITLE
DOTs and resistance check upon application

### ISSUE THREAD

**Issue by lequintal69** [the-hyjal-project/bugtracker#331]
Disclaimer: I did not try that on Hyjal, only on classic. I'm making this report to know if Hyjal team knows more about it than I do. (blizzard would not answer despite my several posts)

How DOTS work now:
- **DOTS are subject to partial resistance on their damage**. But even at 300 resistance, the partial resistance is pretty low
This reason given by a GM on tbc pre patch:  _Previously, dots in general were 1/10th as likely to be resisted as normal spells. That has been fixed, they're now as likely to be resisted as other spells._ 
source:
http://web.archive.org/web/20080601184008/http://forums.worldofwarcraft.com/thread.html?topicId=65457765&pageNo=18&sid=1#348

-**DOTS are NOT subject to resistance check upon application**. 
Did many test with 300 shadow resistance. Lock was pvp hit capped (so only 1% base chance to miss). I did not notice any resist upon application. Obviously all other spells would resist accordingly to known resistance formulas

How DOTS could (should?) work ?
**-DOTS, like other spells, should be subject to magical resistance check upon application.**
In classic right now, spells like sheep, fear, roots are subject to a resistance check upon application AND a heartbeat resistance check on their effect. Now this does not prove it per say, but it seems weird having DOTS bypassing entirely magical resistance check upon application.

So to sum it up, 3 questions:
1/should DOTS be subject to magical resistance check upon application?
2/if so, with which formula? Should the 1/10th rate be applied to that aswell?
3/more generally speaking, should that 1/10th ratio "bug" be introduced for the whole vanilla timeframe ?

**Comment by punker85** [comment]
Partial resistance for PvP DoT auras was added to the game during Wrath of the Lich King expansion. However, I do remember seeing these partial resists of DoTs in an old vanilla video of Twin Emperors battle in Ahn'Qiraj.

**Comment by lequintal69** [comment]
No they werent.
Partial resistance were a thing in vanilla, and they are in classic right now.
https://www.youtube.com/watch?v=SRXUTgrZOvM&feature=youtu.be&t=16m55s
check the combat log:  2 partial resistances there
